tad/schema/github.py

```diff
@@ -0,0 +1,24 @@
+from pydantic import BaseModel, HttpUrl, RootModel
```

Comment:
Should we enforce here that only https is allowed? https://docs.pydantic.dev/latest/api/networks/#pydantic.networks.AnyHttpUrl

Reply:
We have no control over it since github creates these urls